dandi/tests/test_utils.py

```diff
@@ -363,8 +364,14 @@ def test_get_instance_actual_dandi():
         assert inst.api is not None
 
 
+@pytest.mark.redirector
 def test_server_info():
-    r = requests.get(known_instances["dandi"].redirector.rstrip("/") + "/server-info")
+    r = requests.get(
+        os.environ.get(
+            "DANDI_REDIRECTOR_BASE", known_instances["dandi"].redirector
```

Comment:
why don't we override it directly in the `dandi` of the `known_instances`?  this would be the only way to test not only server_info but also any other interaction with the redirector

Reply:
PR closed in favor of switching back to #581.